source/includes/reference/_profiles.md

```diff
@@ -327,3 +327,354 @@ issuerCountry            | Issued by country code. For example "US"    | Text
+     {
+         "id": <shareholder id>,
+         "name": "Jane Doe",
+         "dateOfBirth": "1981-12-07",
+         "countryOfResidenceIso3Code": "usa",
+         "addressFirstLine": "140 Wild Rose Dr.",
+         "postCode": "MI 25887",
+         "ownershipPercentage": 40
+     },
+     {
+         "id": <shareholder id>,
+         "name": "Oliver Wilson",
+         "dateOfBirth": "2017-12-31",
+         "countryOfResidenceIso3Code": "gbr",
+         "addressFirstLine": "22 Baker St",
+         "postCode": "NW1 JA6",
+         "ownershipPercentage": 30
+     }
+]
+```
+Adds new ultimate beneficial owners to the business profile.
+Returns the list of all ultimate beneficial owners associated with the business profile.
+
+### Request (Business)
+**`POST https://api.sandbox.transferwise.tech/v1/profiles/{profileId}/ubos`**
+
+
+## Replace business ultimate beneficial owners
+
+> Example Request:
+
+```shell
+curl -X PUT https://api.sandbox.transferwise.tech/v1/profiles/{profileId}/ubos \
+     -H "Authorization: Bearer <your api token>" \
+     -H "Content-Type: application/json" \
+     -d '[
+             {
+                 "name": "John Doe",
+                 "dateOfBirth": "1982-05-20",
+                 "countryOfResidenceIso3Code": "usa",
+                 "addressFirstLine": "87 West Central St",
+                 "postCode": "MI 48601",
+                 "ownershipPercentage": 30
+             },
+             {
+                 "name": "Jane Doe",
+                 "dateOfBirth": "1981-12-07",
+                 "countryOfResidenceIso3Code": "usa",
+                 "addressFirstLine": "140 Wild Rose Dr.",
+                 "postCode": "MI 25887",
+                 "ownershipPercentage": 70
+             }
+        ]'
+```
+
+> Example Response:
+
+```json
+[
+     {
+         "id": <shareholder id>,
+         "name": "John Doe",
+         "dateOfBirth": "1982-05-20",
+         "countryOfResidenceIso3Code": "usa",
+         "addressFirstLine": "87 West Central St",
+         "postCode": "MI 48601",
+         "ownershipPercentage": 30
+     },
+     {
+         "id": <shareholder id>,
+         "name": "Jane Doe",
+         "dateOfBirth": "1981-12-07",
+         "countryOfResidenceIso3Code": "usa",
+         "addressFirstLine": "140 Wild Rose Dr.",
+         "postCode": "MI 25887",
+         "ownershipPercentage": 70
+     }
+]
+```
+Replaces ultimate beneficial owners in the business profile.
+Returns the list of all ultimate beneficial owners associated with the business profile.
+
+### Request (Business)
+**`PUT https://api.sandbox.transferwise.tech/v1/profiles/{profileId}/ubos`**
+
+
+## Open update window
+
+> Example Request:
+
+```shell
+curl -X POST https://api.sandbox.transferwise.tech/v1/profiles/{profileId}/update-window \
+     -H "Authorization: Bearer <your api token>" 
+```
+
+> Example Response:
+
+```json
+
+```
+
+Opens the update window for updating the business profile information: details, directors, owners.
```

Comment:
This is not just business right? Also this endpoint will only be for banks, maybe we should have separate docs? Not sure how to handle this.

Reply:
Left as-is for now. The endpoint is only referenced in bank section and calling it with non-bank token throws an exception in rest gateway.